README.md

```diff
@@ -14,4 +14,7 @@ GitHub Gems is a data analytics tool designed to track and surface trending open
 - **Data Format**: CSV and JSON formats.
 - **Update Frequency**: Updated daily for real-time insights.
 
+## dbt Lineage Diagram
+![dbt Lineage Diagram](C:\Users\maham\github-stars-pipeline\docs\dbt_dag.png)
```

Comment:
It's best practice to make sure you never push an absolute path to Github, for 2 reasons:
1. This code won't work on my machine (different path to my repo).
2. It's a slight security risk to expose your username and local disk structure.

In this case, `.\docs\dbt_dag.png` should work.
Since you're on window, I'd suggest trying `docs/dbt_dag.png` and if it works, favor it as it's more common to use the Linux format. If it doesn't work, you can keep the forward slashes `\`.